lib/std/Target.zig

```diff
@@ -979,26 +979,38 @@ pub const Cpu = struct {
     };
 
     pub const Arch = enum {
+        alpha,
         arm,
         armeb,
         aarch64,
         aarch64_be,
         aarch64_32,
         arc,
+        arc64,
         avr,
         bpfel,
         bpfeb,
         csky,
         dxil,
         hexagon,
+        hppa,
+        hppa64,
+        kvx,
         loongarch32,
         loongarch64,
         m68k,
+        microblaze,
+        microblazeel
+        microblaze64,
+        microblaze64el,
         mips,
         mipsel,
         mips64,
         mips64el,
+        moxie,
+        moxieel,
         msp430,
+        or1k,
```

Comment:
OpenRISC does specify a 64-bit architecture, but no core designs exist for that yet, and no software supports it. In practice, OpenRISC is basically considered a 32-bit architecture, so not adding a 64-bit variant here yet.